inkstitch.py

```diff
@@ -48,54 +81,41 @@
 
     if running_from_inkscape:
         # process creation of the Bash script - should be done before sys.path is modified, see below in prefere_pip_inkex
-        if ini.getboolean("DEBUG", "create_bash_script", fallback=False):  # create script only if enabled in DEBUG.ini
+        if safe_get(ini, "DEBUG", "create_bash_script", default=False):  # create script only if enabled in DEBUG.toml
             debug_utils.write_offline_debug_script(SCRIPTDIR, ini)
 
         # disable debugger when running from inkscape
-        disable_from_inkscape = ini.getboolean("DEBUG", "disable_from_inkscape", fallback=False)
+        disable_from_inkscape = safe_get(ini, "DEBUG", "disable_from_inkscape", default=False)
         if disable_from_inkscape:
             debug_type = 'none'  # do not start debugger when running from inkscape
 
     # prefer pip installed inkex over inkscape bundled inkex, pip version is bundled with Inkstitch
     # - must be be done before importing inkex
-    prefere_pip_inkex = ini.getboolean("LIBRARY", "prefer_pip_inkex", fallback=True)
+    prefere_pip_inkex = safe_get(ini, "LIBRARY", "prefer_pip_inkex", default=True)
     if prefere_pip_inkex and 'PYTHONPATH' in os.environ:
         debug_utils.reorder_sys_path()
 
-# enabling of debug depends on value of debug_type in DEBUG.ini file
+# enabling of debug depends on value of debug_type in DEBUG.toml file
 if debug_type != 'none':
-    from lib.debug import debug  # import global variable debug - don't import whole module
-    debug.enable(debug_type, SCRIPTDIR, ini)
+    from lib.debug.debugger import init_debugger
+    init_debugger(debug_type, ini)
     # check if debugger is really activated
     debug_active = bool((gettrace := getattr(sys, 'gettrace')) and gettrace())
 
-# warnings are used by some modules, we want to ignore them all in release
-#   - see warnings.warn()
-if running_as_frozen or not debug_active:
-    import warnings
-    warnings.filterwarnings('ignore')
-
-# TODO - check if this is still needed for shapely, apparently shapely now uses only exceptions instead of io.
-#        all logs were removed from version 2.0.0 and above
-#  ---- plan to remove this in future ----
-# import logging # to set logger for shapely
-# from io import StringIO  # to store shapely errors
-# set logger for shapely - for old versions of shapely
-# logger = logging.getLogger('shapely.geos')  # attach logger of shapely
-# logger.setLevel(logging.DEBUG)
-# shapely_errors = StringIO()                # in memory file to store shapely errors
-# ch = logging.StreamHandler(shapely_errors)
-# ch.setLevel(logging.DEBUG)
-# formatter = logging.Formatter('%(name)s - %(levelname)s - %(message)s')
-# ch.setFormatter(formatter)
-# logger.addHandler(ch)
-#  ---- plan to remove this in future ----
+# activate logging for svg
+from lib.debug import debug  # noqa: E402  # import global variable debug - don't import whole module
```

Comment:
Why use `# noqa: E402` instead of moving this import to the top?

Reply:
We need to delay the inkex import. I have added a note.